packages/code_excerpt_updater/test_data/src/dartdoc.md

```diff
@@ -0,0 +1,7 @@
+## API docs
+
+Code block containing triple-backticks:
```

Comment:
I feel this comment is incorrect?

Reply:
You're right, nor is it super useful in that state. It was mostly copied from other similar tests.

I've updated this and the other new test data files with more detailed comments describing the purpose of the test case. Thanks for calling this out!